content/publication_resources/bibtex/lahti.bib

```diff
@@ -1,3 +1,19 @@
+@Article{Erawijantari2023,
+  author =	 {Pande Putu Erawijantari and Ece Kartal and José Liñares-Blanco and Teemu D. Laajala and 
+  Lily Elizabeth Feldman and The FINRISK Microbiome DREAM Challenge and ML4Microbiome Communities and 
+  Pedro Carmona-Saez and Rajesh Shigdel and Marcus and Joakim Claesson and Randi Jacobsen Bertelsen and 
+  David Gomez-Cabrero and Samuel Minot and Jacob Albrecht and Verena Chung and Michael Inouye and 
+  Pekka Jousilahti and Jobst-Hendrik Schultz and Hans-Christoph Friederich and Rob Knight and Veikko Salomaa and 
+  Teemu Niiranen and Aki S. Havulinna and Julio Saez-Rodriguez and Rebecca T. Levinson and Leo Lahti},
+  title =	 {Microbiome-based risk prediction in incident heart failure: a community challenge},
+  journal =	 {{medRxiv}},
+  pubstate =	 {preprint},
+  year =	 2023,
+  month =	 10,
+  doi =		 {10.1101/2023.10.12.23296829},
+  keywords =	 {bioscience}
+}
```

Comment:
Also add tag on preprint, check other bibtext entries for an example (search "preprint")